docs/integrations/tax.mdx

```diff
@@ -33,14 +33,15 @@ Once your tax provider configuration is ready, we'll let you know through email.
 | Platform availability | Required | All stores, Limited availability | Whether your tax provider will be available to all stores or a limited number of stores. | `Limited availability` |
 | Supported / unsupported countries | Required | ISO 3166-1 alpha-2 | Comma separated ISO 3166-1 alpha-2 country codes for supported countries. You can also select to support to all countries, or all countries except a list of unsupported countries. | `US,CA,GB,FR,AU,NZ` |
 | **URLs** | | | | |
-| <sup>+</sup>Estimate URL | Required | URL | URL BigCommerce should use for Tax Provider API estimate requests. | `https://sampletax.example.com/tax/estimate` |
-| <sup>+</sup>Commit URL | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/commit` |
-| <sup>+</sup>Adjust URL | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/adjust` |
-| <sup>+</sup>Void URL | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/void` 
+| Estimate URL <sup>1</sup> | Required | URL | URL BigCommerce should use for Tax Provider API estimate requests. | `https://sampletax.example.com/tax/estimate` |
+| Commit URL <sup>1</sup><sup>2</sup> | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/commit` |
+| Adjust URL <sup>1</sup><sup>2</sup> | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/adjust` |
+| Void URL <sup>1</sup><sup>2</sup> | Optional | URL | URL BigCommerce should use for Tax Provider API quote requests. | `https://sampletax.example.com/doc/void`
 | **Testing** | | | | |
 | Partner sandbox store domain | Required | Domain name | Share your partner sandbox store for testing purposes prior to launching your tax provider. Learn how to [create a partner sandbox store](/docs/start/about/sandboxes). | `https://sampletax-test-store.mybigcommerce.com/` |
 
-<sup>+</sup> You have the option of supporting a URL that merchants can customize. See [Tax Profile](#tax-profile-optional).
+<sup>1</sup> You have the option of supporting a URL that merchants can customize. See [Tax Profile](#tax-profile-optional).
+<sup>2</sup> URL is only called when the **Submit Order Data** provider setting is enabled. See [Document Submission](#document-submission).
```

Comment:
@jvpcode Great suggestion sir, thanks. I've taken the approach of adding a superscript point to highlight this - WDYT? 🙏 Cheers.

Reply:
![applause](https://github.com/user-attachments/assets/c7da0168-4e31-4781-9436-4fdb8e0b7304)